packages/@react-aria/listbox/src/useOption.ts

```diff
@@ -127,15 +121,16 @@ export function useOption<T>(props: AriaOptionProps, state: ListState<T>, ref: R
     optionProps['aria-setsize'] = getItemCount(state.collection);
   }
 
-  let {itemProps, isPressed} = useSelectableItem({
+  let {itemProps, isPressed, hasAction, allowsSelection} = useSelectableItem({
     selectionManager: state.selectionManager,
     key,
     ref,
     shouldSelectOnPressUp,
-    allowsDifferentPressOrigin: shouldSelectOnPressUp,
+    allowsDifferentPressOrigin: shouldSelectOnPressUp && shouldFocusOnHover,
```

Comment:
standalone list boxes should not allow different press origin, only ones within overlays like picker, combobox, etc.